server/lib/currency.js

```diff
@@ -37,21 +37,18 @@ if (!get(config, 'fixer.accessKey') && !['staging', 'production'].includes(confi
 export function getFxRate(fromCurrency, toCurrency, date = 'latest') {
   debug('>>> getFxRate for ', date, fromCurrency, toCurrency);
 
-  if (!get(config, 'fixer.accessKey')) {
+  if (fromCurrency === toCurrency) {
+    return Promise.resolve(1);
+  } else if (!fromCurrency || !toCurrency) {
```

Comment:
Shouldn't we throw exception for that?

Reply:
Not sure, I'm worried that there's some edge-case here that I'm not aware.
I think we should keep the same logic.

Reply:
I missed it was just moved from somewhere else. It's fine then.